app/lib/katello/resources/cdn/katello_cdn.rb

```diff
@@ -2,8 +2,11 @@ module Katello
   module Resources
     module CDN
       class KatelloCdn < CdnResource
+        attr_reader :organization_label, :content_view_label, :lifecycle_environment_label
```

Comment:
Sorry if I missed it but if these don't need to be accessed from outside the instance of this class, I prefer using the instance variables.